package.json

```diff
@@ -19,6 +19,15 @@
 		"index.js",
 		"index.d.ts"
 	],
+	"os": [
+		"android",
+		"darwin",
+		"freebsd",
+		"linux",
+		"openbsd",
+		"sunos",
+		"win32"
+	],
```

Comment:
I don't use `os` in any packages. It's buggy and I don't want to prevent a consumer of this package to support different platforms just because they depend on `internal-ip`. Runtime checks are better with `optionalDependencies` is a better choice.

But it doesn't matter. `default-gateway` has `os`, so this package doesn't need it anyway.

Reply:
Okay, `os` removed.